File: 1880-check-if-word-equals-summation-of-two-words/1880-check-if-word-equals-summation-of-two-words.py
class Solution(object):
    def isSumEqual(self, firstWord, secondWord, targetWord):
        def word_to_number(word):
            value = 0
            for ch in word:
                digit = ord(ch) - ord('a')
                value = value*10 + digit
            return value

        num1 = word_to_number(firstWord)
        num2 = word_to_number(secondWord)
        num3 = word_to_number(targetWord)

        return num1+num2 == num3

File: 1880-check-if-word-equals-summation-of-two-words/1880-check-if-word-equals-summation-of-two-words.java
class Solution 
{
    public boolean isSumEqual(String firstWord, String secondWord, String targetWord) 
    {
        int num1 = wordToNumber(firstWord);
        int num2 = wordToNumber(secondWord);
        int num3 = wordToNumber(targetWord);
        return num1 + num2 == num3;
    }

    public int wordToNumber(String word) 
    {
        int value = 0;
        for (int i = 0; i < word.length(); i++) 
        {
            int digit = word.charAt(i) - 'a';
            value = value * 10 + digit;
        }
        return value;
    }
}